### ISSUE TITLE
Plugin doesn't works with GLPI 9.5.3

### ISSUE THREAD

**Issue by GoodFeel91** [InfotelGLPI/badges#27]
Hello,

I used to use the Badges plugin to manage my access badges. This plugin was working very well on my GLPI version 9.3/9.4.

I updated my instance a little more than a month ago to the 9.5 version of GLPI and integrated the latest version of the badge plugin (2.6.0) which is supposed to be compatible with the 9.5.x version. I can activate the plugin and access the available badges, but when I want to assign it, by clicking on "Add to cart" or "Send", nothing happens. Also when I click on "Cancel", the buttons seem inactive:

I tried to remove/reinstall the plugin but same thing, there is nothing to do.

------------
* Version of the plugin : 2.6.0

* Version of your GLPI : 9.5.3

* Steps to reproduce (which actions have you made) : When I click on "Add to cart" or "Send" or "Cancel", nothing happens.

* Expected result : booking validation

* Actual result : Nothing

* URL of the page : https://mydomain.tld/plugins/badges/front/wizard.form.php?action=badgerequest# (this "#" is added when I click on the button (add to card, send or cancel)

* Screenshot of the problem (if pertinent) : https://i.gyazo.com/a58069cea7c85de9aa8a0814e48eaa08.png

Thank you for your help.

**Comment by tsmr** [comment]
Hello? You have install the plugin from Marketplace or directly ?

**Comment by GoodFeel91** [comment]
Hello,

I installed directly from source on Github : https://github.com/InfotelGLPI/badges/releases

Thank you.

**Comment by tsmr** [comment]
You can try this fix ? https://github.com/InfotelGLPI/badges/commit/8b8d2582512240331d8b422c51174c70f7bfabc8

**Comment by GoodFeel91** [comment]
I modified this 3 files (on marketplace and source) and I have the same behaviour, nothing happened.

**Comment by tsmr** [comment]
Can you activate console on your browser for see errors ?

**Comment by GoodFeel91** [comment]
Yes, I have an error when I reach the page "Badge" :

https://i.gyazo.com/ae7381a94681614afd8a92df4038d86f.png

I have an other error when I click on the buttons :

https://i.gyazo.com/f9040654029b7826ecc3dfbb0637037a.png

Thank you.

**Comment by tsmr** [comment]
Have you tried to clean browser cache ?

**Comment by GoodFeel91** [comment]
Yes, I even tried on mulitple browsers (Firefox, Chrome, Edge) and on private navigation and I reproduce it.

I also tried to install the last GLPI version (9.5.4) and I reproduce it. When I downgrade on GLPI 9.4/9.3 with Badge 2.5.0 (because 2.6.0 is not compatible), it's works fine.

**Comment by tsmr** [comment]
Can you try to add into setup.php 

$PLUGIN_HOOKS["javascript"]['badges']     = ["/plugins/badges/badges.js"];

after 

$PLUGIN_HOOKS['add_javascript']['badges'][] = 'badges.js';

?

**Comment by GoodFeel91** [comment]
I add this line but same effect. I tried to uncomment the line $PLUGIN_HOOKS['add_javascript']['badges'][] = 'badges.js'; because it's was commented on my setup.php file, but both didn't work.

I have some errors on console :

![2021-03-19 11_18_49-](https://user-images.githubusercontent.com/78422494/111766133-780c8580-88a5-11eb-9b74-335dd9433919.png)
![2021-03-19 11_20_31-](https://user-images.githubusercontent.com/78422494/111766137-78a51c00-88a5-11eb-868d-55f5970ddb12.png)
![2021-03-19 11_20_15-](https://user-images.githubusercontent.com/78422494/111766138-793db280-88a5-11eb-93a9-c50d9693cf2c.png)

**Comment by skanigins** [comment]
Having the same issue as described above
